cloudinary-core/src/main/java/com/cloudinary/Uploader.java

```diff
@@ -323,40 +323,130 @@ public Map explode(String public_id, Map options) throws IOException {
         return callApi("explode", params, options, null);
     }
 
+
```

Comment:
This line looks redundant, and the comment below is very confusing, please move it to the right place in the doc string